flight/Modules/GPS/UBX.c

```diff
@@ -251,6 +252,7 @@ static void parse_ubx_nav_velned (const struct UBX_NAV_VELNED *velned, GPSPositi
 			GpsVelocity.North	= (float)velned->velN/100.0f;
 			GpsVelocity.East	= (float)velned->velE/100.0f;
 			GpsVelocity.Down	= (float)velned->velD/100.0f;
+			GpsVelocity.Accuracy	= (float)velned->sAcc/100.0f;
```

Comment:
according to ```struct UBX_NAV_SOL``` this starts at cm/s accuracy. you are trying to convert to 100µm/s ?